snuba/api.py

```diff
@@ -241,8 +245,10 @@ def dataset_query(dataset, body, timer):
     )
 
 
-def format_query(dataset, body, table, prewhere_conditions, final) -> str:
+def format_query(dataset, request: Request, table, prewhere_conditions, final) -> str:
     """Generate a SQL string from the parameters."""
+    body = request.body
```

Comment:
See note in the description — this will be next, and I don't think it'll actually be as complicated as I expected.